File: frontend/src/data-services/api-services/models/queues/queue-overview-dto.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

export interface QueueOverviewDto {
    /**
     * Queue id
     */
    [key: string] : {
        lockedItemsCount: number,
        nearToSlaCount: number,
        nearToTimeoutCount: number
    }
}

File: backend/queues/src/main/java/com/griddynamics/msd365fp/manualreview/queues/model/dto/QueueOverviewDTO.java
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

package com.griddynamics.msd365fp.manualreview.queues.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class QueueOverviewDTO {

    private long lockedItemsCount;
    private long nearToSlaCount;
    private long nearToTimeoutCount;

}
